### ISSUE TITLE
Websocket randomly disconnecting

### ISSUE THREAD

**Issue by popatop15** [nguyenquyhy/DiscordBridge#33]
https://gyazo.com/0ac90cea007a26a5917bbc71fad7aac6

For some reason I am getting console spammed about every few seconds with a websocket disconnected for an unknown reason error message.

**Comment by Mohron** [comment]
I also experience this issue quite regularly but have noticed that it never seems to affect the function. It is highly annoying though. It's likely an issue with the Javacord APIs handling of disconnections. A `discord reload` clears this issue up until there's a new websocket error that begins the spamming again.

Any ideas @nguyenquyhy?

**Comment by nguyenquyhy** [comment]
Yes I am trying to solve this issue. Basically when there is a disconnection, the plugin will try to reconnect. However, for some reasons, even after successfully reconnected, it still attempts to reconnect again, which makes the whole thing fall into a infinite retry loop.

I am trying to fix this in the next update.

**Comment by BtoBastian** [comment]
Hey,
Javacord 2.0.13 has been released which should fix this bug.
I'm sorry for inconvenience. 
https://github.com/BtoBastian/Javacord/commit/0069a026e29f0be395fc7a020337c25ac0201691

**Comment by Sitherious** [comment]
woot :)

**Comment by nguyenquyhy** [comment]
Please try the latest 2.3.0 version and let me know if you still get this error 😄 .
